dockerfile

```diff
@@ -0,0 +1,47 @@
+# Use python:buster image as base
+FROM python:buster as base
+
+# Update pip
+RUN apt-get update && pip install --upgrade pip
+RUN apt-get install -y curl
+
+# Install Poetry and add to path
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH = "${PATH}:/root/.poetry/bin"
+
+FROM base as production
+
+# Expose Port 8000
+EXPOSE 8000
+
+# Copy code accross
+COPY . /usr/DevOps-Course-Starter
+
+WORKDIR /usr/DevOps-Course-Starter
+
+# Install poetry dependencies 
+RUN poetry install
```

Comment:
You could change this to `poetry install --no-dev` to skip installing things like selenium or pytest that you won't ever need in your production image.